src/doc_builder/autodoc.py

```diff
@@ -335,3 +335,44 @@ def autodoc(object_name, package, methods=None, return_anchors=False, page_info=
     documentation = '<div class="docstring">\n' + documentation + "</div>\n"
 
     return (documentation, anchors) if return_anchors else documentation
+
+
+def resolve_links_in_text(text, package, mapping, page_info):
+    """
+    Resolve links of the form [`SomeClass`] to the link in the documentation to `SomeClass`.
+
+    Args:
+    - **text** (`str`) -- The text in which to convert the links.
+    - **package** (`types.ModuleType`) -- The package in which to search objects for.
+    - **mapping** (`Dict[str, str]`) -- The map from anchor names of objects to their page in the documentation.
+    - **page_info** (`Dict[str, str]`) -- Some information about the page.
+    """
+    package_name = page_info.get("package_name", package.__name__)
+    version = page_info.get("version", "master")
+    language = page_info.get("language", "en")
+
+    prefix = f"/docs/{package_name}/{version}/{language}/"
```

Comment:
Probably not important, but this won't work on Windows (I know it was here before!)

Reply:
It's an URL not a local path. I think they work the same way on Windows ;-)